### DISCUSSION TITLE
New reviewer for C/C++

### DISCUSSION THREAD

**Discussion by Ashborn-SM** [MakeContributions/DSA#474] [General]
@ming-tsai, @Arsenic-ATG, @UG-SEP , @aayushjain7 is it possible for me to become a reviewer for C/C++ as well? what do you guys think?

**Comment by ming-tsai** [comment]
> @ming-tsai, @Arsenic-ATG, @UG-SEP , @aayushjain7 is it possible for me to become a reviewer for C/C++ as well? what do you guys think?

Ok, Added on the readme, please see the Review Rule on discord or this [discussion](https://github.com/MakeContributions/DSA/discussions/232)
